package.json

```diff
@@ -50,6 +72,8 @@
   "devDependencies": {
     "bower": "^1.7.1",
     "tslint": "^3.14.0",
+    "electron-builder": "^5.22.2",
+    "electron-prebuilt": "^1.3.2",
```

Comment:
Please do not move electron-prebuilt from dependency to dev dependency.  npm package won't work if Electron binary is in dev dependency.